tools/build_defs/repo/git_worker.bzl

```diff
@@ -196,8 +196,11 @@ def _git_maybe_shallow(ctx, git_repo, command, *args):
     return _execute(ctx, git_repo, start + args_list)
 
 def _execute(ctx, git_repo, args):
```

Comment:
I moved the passing of `git` to be implied in this function as its intended to run `git` commands anyway. This allows me to add `-c` which must come directly after `git` without searching through the `args` we are given